I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/advanced_scripts/definitions/definitions.rpy", line 83, in script
    define reply = renpy.random.choice(replies)
  File "game/advanced_scripts/definitions/definitions.rpy", line 83, in script
    define reply = renpy.random.choice(replies)
  File "game/advanced_scripts/definitions/definitions.rpy", line 83, in <module>
    define reply = renpy.random.choice(replies)
NameError: name 'replies' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/bootstrap.py", line 331, in bootstrap
    renpy.main.main()
  File "renpy/main.py", line 560, in main
    renpy.game.context().run(node)
  File "game/advanced_scripts/definitions/definitions.rpy", line 83, in script
    define reply = renpy.random.choice(replies)
  File "game/advanced_scripts/definitions/definitions.rpy", line 83, in script
    define reply = renpy.random.choice(replies)
  File "renpy/ast.py", line 2213, in execute
    self.set()
  File "renpy/ast.py", line 2224, in set
    value = renpy.python.py_eval_bytecode(self.code.bytecode)
  File "renpy/python.py", line 2269, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/advanced_scripts/definitions/definitions.rpy", line 83, in <module>
    define reply = renpy.random.choice(replies)
NameError: name 'replies' is not defined

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
Doki Doki Detective Club 1.1.2
Tue Aug  6 19:06:13 2024
